keps/sig-scheduling/4815-dra-partitionable-devices/README.md

```diff
@@ -3117,12 +1420,29 @@ No.
 
 ###### Will enabling / using this feature result in increasing size or count of the existing API objects?
 
-Yes and No. With the extensions proposed in this KEP, individual
+Yes. With the extensions proposed in this KEP, individual
 `ResourceSlices` have additional fields available to them, thus increasing
-their overall signature. However, the purpose of the new `Mixins` abstraction
-is to actually *reduce* the footprint of these objects, not increase them. As
-such, we expect the size of thes eobjects to actually decrease, not increase,
-but it is ultimately up to how 3rd party vendors decide to use them.
+their overall signature. A separate Mixins feature was previously part of this
+change, but will now be handled separately. It will add features that let
+users define devices in a more compact way and thereby reducing the size of API
+objects. But it is ultimately up to how 3rd party vendors decide to use them.
+
+To limit the size of the `ResourceSlice` objects we have implemented several limits that
+is enforced by validation:
+* The maximum number of counters in a single `ResourceSlice` is limited to 32.
+* The maximum number of devices in a single `ResourceSlice` is limited to 128.
+
+For each `Device` the following limits are enforced:
+* The maximum number of attributes and capacity in a single `Device` is limited to 32.
+* The maximum number of consumed counters in a single `Device` is limited to 32.
+
+Across all `Device` objects in a `ResourceSlice`, the following limits are enforced:
+* The total number of consumed counters across all `Device` objects in a single `ResourceSlice`
+  is limited to 1024.
+
+The `ResourceSlice`-wide limits on fields within the `Device` object is used to allow users
+to decide whether to have few devices with many properties or many devices with few properties.
```

Comment:
How much does the worst-case ResourceSlice size grow now with these limits?

We traditionally haven't gone into those details in KEPs, but I think we can answer this already and it would be good to document.

Reply:
I've added the data about the increase in the worst-case ResourceSlice size.